enqueue/enqueue-bundle/0.3/Makefile

```diff
@@ -0,0 +1,3 @@
+consume:
+	@test -f bin/console && bin/console enqueue:consume --setup-broker || echo "cannot consume messages (needs symfony/console)"
+.PHONY: consume
```

Comment:
I think this can be removed as well. We don't want to wrap all Symfony CLI commands as make tasks. We are using Makefile tasks for when this is not practical or not possible to use the Symfony CLI.